qds_sdk/qubole.py

```diff
@@ -87,3 +89,8 @@ def agent(cls, version=None):
           cls.cached_agent = Connection(cls._auth, cls.rest_url, cls.skip_ssl_cert_check)
 
         return cls.cached_agent
+
+    @classmethod
+    def get_cloud(cls):
+        return Connection(cls._auth, cls.baseurl.rstrip("/").replace(cls.baseurl.rstrip("/").split("/")[-1], "")
```

Comment:
I think getting details from about is fine which ultimately read the constants of the environment i.e from creds.yml . Also all the accounts of the production link will have same cloud provider which is not the right way to save it .I have already merged changes related to this on tapp master. https://bitbucket.org/qubole/qbol/commits/8216e4a939672f5bf1187e0087e5850363322a09?at=master